pandas/tests/io/parser/common/test_read_errors.py

```diff
@@ -148,7 +166,14 @@ def test_error_bad_lines(all_parsers):
     data = "a\n1\n1,2,3\n4\n5,6,7"
 
     msg = "Expected 1 fields in line 3, saw 3"
-    with pytest.raises(ParserError, match=msg):
+    ex_type = ParserError
```

Comment:
A bit last minute, but I'm wondering if it makes sense to catch the ``ArrowInvalid`` error raised, and re-raise it with our own ``ParserError` to keep things consistent.

Reply:
That approach could be applied as well. I'm assuming it's alright to keep the standard error text that's passed up within ArrowInvalid by pyarrow?